x-pack/plugins/security_solution/cypress/screens/alerts_detection_rules.ts

```diff
@@ -66,8 +68,18 @@ export const SHOWING_RULES_TEXT = '[data-test-subj="showingRules"]';
 
 export const SORT_RULES_BTN = '[data-test-subj="tableHeaderSortButton"]';
 
-export const THREE_HUNDRED_ROWS = '[data-test-subj="tablePagination-300-rows"]';
-
 export const RULE_AUTO_REFRESH_IDLE_MODAL = '[data-test-subj="allRulesIdleModal"]';
 
 export const RULE_AUTO_REFRESH_IDLE_MODAL_CONTINUE = '[data-test-subj="allRulesIdleModal"] button';
+
+export const rowsPerPageSelector = (count: number) =>
+  `[data-test-subj="tablePagination-${count}-rows"]`;
+
+export const FIVE_ROWS = rowsPerPageSelector(5);
+export const THREE_HUNDRED_ROWS = rowsPerPageSelector(300);
+
+export const pageSelector = (pageNumber: number) =>
+  `[data-test-subj="pagination-button-${pageNumber - 1}"]`;
+
+export const FIRST_PAGE_SELECTOR = pageSelector(1);
+export const SECOND_PAGE_SELECTOR = pageSelector(2);
```

Comment:
Not sure if constants like `FIVE_ROWS` are needed. Unless a constant is a kind of default for the majority of tests, I'd keep the function and get rid of the constants.

Reply:
FYI cleaned this up in a separate commit.